app/uk/gov/hmrc/helptosavefrontend/views/introduction/navigation.scala.html

```diff
@@ -14,6 +14,9 @@
  * limitations under the License.
  *@
 
+@import cats.instances.string._
+@import cats.syntax.eq._
+
```

Comment:
can we disable warts for all views so that views cane be made free from cats ? :) 
not a big deal though..

Reply:
since there's scala code in the views I think it's probably a good idea to keep it there so that the code quality is consistent across the whole code base - similarly for the selenium tests as well